runtime/static_config.go

```diff
@@ -115,11 +115,33 @@ func NewStaticConfigOrDie(
 	return config
 }
 
-// MustGetBoolean returns the value as a boolean or panics.
-func (conf *StaticConfig) MustGetBoolean(key string) bool {
+func (conf *StaticConfig) checkConfDestroyed(key string) {
 	if conf.destroyed {
-		panic(errors.Errorf("Cannot get(%s) because destroyed", key))
+		panic(errors.Errorf("Cannot get (%s) because destroyed", key))
 	}
+}
+
+// GetOrDefaultBoolean returns the boolean value for a key or returns a default value
+func (conf *StaticConfig) GetOrDefaultBoolean(key string, defaultBool bool) bool {
```

Comment:
why do you need this method? All the exiting use cases rely on `ContainsKey` method.

Reply:
I initially created it as a convenience method, but that's a good point. I removed this and refactored the logic to rely on `ContainsKey` instead for consistency.